[2023-12-28 14:56:22,819] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: OpenWeather_ETL2.conexion_BD scheduled__2023-12-27T00:00:00+00:00 [queued]>
[2023-12-28 14:56:22,853] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: OpenWeather_ETL2.conexion_BD scheduled__2023-12-27T00:00:00+00:00 [queued]>
[2023-12-28 14:56:22,854] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-12-28 14:56:22,856] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2023-12-28 14:56:22,856] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-12-28 14:56:22,892] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): conexion_BD> on 2023-12-27 00:00:00+00:00
[2023-12-28 14:56:22,903] {standard_task_runner.py:52} INFO - Started process 1083 to run task
[2023-12-28 14:56:22,913] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'OpenWeather_ETL2', 'conexion_BD', 'scheduled__2023-12-27T00:00:00+00:00', '--job-id', '13', '--raw', '--subdir', 'DAGS_FOLDER/ETL_OpenWeather2.py', '--cfg-path', '/tmp/tmpqyt66x9i', '--error-file', '/tmp/tmpkspxgc_z']
[2023-12-28 14:56:22,917] {standard_task_runner.py:80} INFO - Job 13: Subtask conexion_BD
[2023-12-28 14:56:23,079] {task_command.py:371} INFO - Running <TaskInstance: OpenWeather_ETL2.conexion_BD scheduled__2023-12-27T00:00:00+00:00 [running]> on host 98dcb739da71
[2023-12-28 14:56:23,182] {logging_mixin.py:115} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:202 AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
[2023-12-28 14:56:23,275] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Furtado_Julio
AIRFLOW_CTX_DAG_ID=OpenWeather_ETL2
AIRFLOW_CTX_TASK_ID=conexion_BD
AIRFLOW_CTX_EXECUTION_DATE=2023-12-27T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-12-27T00:00:00+00:00
[2023-12-28 14:56:23,277] {logging_mixin.py:115} INFO - Conectandose a la DB en la fecha: 2023-12-27 0
[2023-12-28 14:56:24,562] {logging_mixin.py:115} INFO - <connection object at 0x7f00a84e5ad0; dsn: 'user=juliocfurtado93_coderhouse password=xxx dbname=data-engineer-database host=data-engineer-cluster.cyhh5bfevlmn.us-east-1.redshift.amazonaws.com port=5439', closed: 0>
[2023-12-28 14:56:24,563] {logging_mixin.py:115} INFO - Conexion a RedShift exitosa!
[2023-12-28 14:56:24,565] {python.py:173} INFO - Done. Returned value was: None
[2023-12-28 14:56:24,609] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=OpenWeather_ETL2, task_id=conexion_BD, execution_date=20231227T000000, start_date=20231228T145622, end_date=20231228T145624
[2023-12-28 14:56:24,693] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-12-28 14:56:24,829] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
